Dockerfile

```diff
@@ -1,11 +1,20 @@
-FROM golang:1.9
+# Builder image
+FROM golang:1.9 as builder
 
 COPY . /go/src/github.com/pingcap/tidb
 
-RUN cd /go/src/github.com/pingcap/tidb && \
-    make && \
-    mv bin/tidb-server /tidb-server && \
-    make clean
+WORKDIR /go/src/github.com/pingcap/tidb/
+
+RUN make
+
+RUN chmod +x /go/src/github.com/pingcap/tidb/bin/tidb-server
+
+# Executable image
+FROM scratch
+
+COPY --from=builder /go/src/github.com/pingcap/tidb/bin/tidb-server /tidb-server
+
+WORKDIR /
 
 EXPOSE 4000
 
```

Comment:
I think ENTRYPOINT should be specified in the executable image.

Reply:
It’s not changed https://github.com/im-kulikov/tidb/blob/cd08d16bc27611da560bcd2421b723b47f00e085/Dockerfile#L21

Reply:
Sorry, I missed it .